libbeat/outputs/elasticsearch/client_integration_test.go

```diff
@@ -85,15 +81,15 @@ func testPublishEvent(t *testing.T, index string, cfg map[string]interface{}) {
 	output, client := connectTestEsWithStats(t, cfg, index)
 
 	// drop old index preparing test
-	client.conn.Delete(index, "", "", nil)
+	_, _, _ = client.conn.Delete(index, "", "", nil)
 
-	batch := outest.NewBatch(beat.Event{
+	batch := encodeBatch(client, outest.NewBatch(beat.Event{
```

Comment:
This linter error (and the similar ones below) is mistaken, there is no typecheck error here.

What's worse, if I add a `nolint` directive to skip it, it gives an error because the directive is "unused", so something is maybe wrong with the linter config...